tests/conftest.py

```diff
@@ -19,6 +30,88 @@ def config():
     return Config()
 
 
+@pytest.fixture
+def ecs_client(mock_ecs_cluster, mock_ecs_task_definition, mock_ecs_network_config):
+    return ECSClient(
+        cluster=f"arn:aws:ecs:us-east-1:{ACCOUNT_ID}:cluster/mock-sapinvoices-ecs-test",
+        task_definition=f"arn:aws:ecs:us-east-1:{ACCOUNT_ID}:task-definition/mock-sapinvoices-test:1",
+        network_configuration=mock_ecs_network_config,
+    )
+
+
+@pytest.fixture
+def mock_boto3_ecs_client():
+    with patch("webapp.utils.aws.ecs.boto3.client") as mock_client:
+        yield mock_client.return_value
+
+
+@pytest.fixture
+def mock_ecs_cluster():
+    """See https://github.com/getmoto/moto/blob/master/tests/test_ecs/test_ecs_boto3.py."""
+    with mock_aws():
+        ecs = boto3.client("ecs", region_name=AWS_DEFAULT_REGION)
+        ecs.create_cluster(clusterName="mock-sapinvoices-ecs-test")
+        yield
+
+
+@pytest.fixture
+def mock_ecs_task_definition():
+    """See https://github.com/getmoto/moto/blob/master/tests/test_ecs/test_ecs_boto3.py."""
+    with mock_aws():
+        ecs = boto3.client("ecs", region_name=AWS_DEFAULT_REGION)
+
+        # Registering with minimal definition
+        definition = {
+            "family": "mock-sapinvoices-test",
+            "containerDefinitions": [
+                {
+                    "name": "mock-sapinvoices-ecs-test",
+                    "image": "mock-sapinvoices-test:latest",
+                    "memory": 400,
+                }
+            ],
+        }
+        ecs.register_task_definition(**definition)
+        yield
+
+
+@pytest.fixture
+def mock_ecs_network_config():
+    return {
+        "awsvpcConfiguration": {
+            "securityGroups": ["sg-abc123"],
+            "subnets": ["subnet-abc123"],
+        }
+    }
+
+
+@pytest.fixture
+def mock_ecs_task_state_transitions(
+    mock_ecs_cluster, mock_ecs_task_definition, mock_ecs_network_config
+):
+    """See https://github.com/getmoto/moto/blob/master/tests/test_ecs/test_ecs_boto3.py."""
+    state_manager.set_transition(
+        model_name="ecs::task", transition={"progression": "manual", "times": 1}
+    )
+
+    with mock_aws():
+        ecs = boto3.client("ecs", region_name=AWS_DEFAULT_REGION)
+        response = ecs.run_task(
+            cluster="mock-sapinvoices-ecs-test",
+            launchType="FARGATE",
+            networkConfiguration=mock_ecs_network_config,
+            overrides={},
+            taskDefinition="mock-sapinvoices-test:1",
+        )
+        yield response["tasks"][0]["taskArn"]
+
+
+@pytest.fixture
+def ecs_run_task_response_success():
+    with open("tests/fixtures/ecs_run_task_response_success.json") as file:
+        return json.loads(file.read())
```

Comment:
I initially tried to use the `boto3.ECS.client.run_task` mocked output from `moto` but a lot of the response parameters were excluded. I opted to create a fixture by creating a JSON file of the de-identified output from a test run.